src/source/Include_i.h

```diff
@@ -108,13 +109,31 @@ typedef struct {
     BOOL isPointToPoint;
 } KvsIpAddress, *PKvsIpAddress;
 
-#define IS_IPV4_ADDR(pAddress) ((pAddress)->family == KVS_IP_FAMILY_TYPE_IPV4)
```

Comment:
Don't recommend deleting this since it's a publicly-accessible item (breaking change)